zstacklib/zstacklib/utils/gpu.py

```
@@ -672,9 +673,222 @@ def get_enflame_gpu_info_cmd():
+
+    Returns:
+        bool: True if device was processed (is a GPU), False otherwise
+    """
+    if not pci_device_to:
+        return False
+
+    from zstacklib.utils.pci import normalize_pci_address
+
+    gpu_info_map = context.gpu_info_map or {}
+    pci_device_mapper = context.pci_device_mapper or {}
+    opaque = context.opaque
```

Ruff (0.14.14)

Local variable `opaque` is assigned to but never used

Remove assignment to unused variable `opaque`

(F841)